README.md

```diff
@@ -1,2 +1,18 @@
 # Faros Events CLI
 CLI for reporting events to Faros platform
+
+## Example Emits
```

Comment:
Lets do each example in it's own section. I.e.

## Examples

### Send Full (Build + Deployment) event 
```sh
./faros_emit.sh full \
   ...
```
### Send Build event
```sh
./faros_emit.sh build \
   ...
```
// etc

Reply:
For example see - https://github.com/faros-ai/terraform-aws-faros#usage-examples